seqr/views/apis/individual_api.py

```diff
@@ -35,6 +32,13 @@
 __AFFECTED_TO_EXPORTED_VALUE['U'] = ''
 
 
+class ErrorsWarningsException(Exception):
```

Comment:
Would go with something like `ParseException` to indicate error context/cause similar to built-in exceptions